### PR TITLE
RRD-7: Tab navigation

### PR DESCRIPTION
**Pull request by ignacioaristo**

### CODE REVIEW

**Review by julianv97 — CHANGES_REQUESTED** [review]
I think what you ran a `npm` command and create a `package-json.lock`, delete it

File: src/screens/login/index.tsx [hunk 1]
```diff
@@ -50,10 +51,7 @@ const Login = ({navigation}: any) => {
         error={errors.password}
         type="password"
       />
-
-      <Button onPress={handleSubmit(onSubmit)} w={'50%'}>
-        Login
-      </Button>
+      <Button text="Login" handleSubmit={handleSubmit} onSubmit={onSubmit} />
```
Comment (julianv97) [inline src/screens/login/index.tsx]: We should not manage handleSubmit and onSubmit by separate, you can create an unique prop called `handleSubmit` and then when you use the Button, you can pass the prop like this: `handleSubmit={handleSubmit(onSubmit)}`

**Review by julianv97 — CHANGES_REQUESTED** [review]

File: src/screens/profile/index.tsx [hunk 2]
```diff
@@ -1,12 +1,73 @@
...
+          uri: 'https://images.unsplash.com/photo-1603415526960-f7e0328c63b1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
+        }}
+        size="2xl">
+        NB
+        <Avatar.Badge
+          bg={DATA_MOCK.status === 'Starter' ? 'green.500' : 'red.500'}
+        />
+      </Avatar>
+      <View style={styles.infoContainer}>
+        <View style={styles.infoRow}>
```
Comment (julianv97) [inline src/screens/profile/index.tsx]: We can create a FlatList to not repeat each `infoRow`, where each renderItem is going to return: `<View style={styles.infoRow}>
          <Text style={styles.infoRowTitle}>{title}</Text>
          <Text style={styles.infoRowText}>{item.lastName}</Text>
        </View>`

**Review by julianv97 — CHANGES_REQUESTED** [review]

File: src/screens/login/index.tsx (hunk 1 above)
Comment (ignacioaristo) [reply src/screens/login/index.tsx]: Done.

File: src/screens/profile/index.tsx (hunk 2 above)
Comment (ignacioaristo) [reply src/screens/profile/index.tsx]: Done.

**Review by julianv97 — APPROVED** [review]

**Pull request merged**